kolibri/core/assets/src/utils/browserInfo.js

```diff
@@ -69,3 +69,16 @@ export const isTouchDevice =
   'ontouchstart' in window ||
   window.navigator?.maxTouchPoints > 0 ||
   window.navigator?.msMaxTouchPoints > 0;
+
+function handlePointerDown(event) {
+  if (event.pointerType === 'mouse') {
+    localStorage.setItem('mouseUsed', 'true');
+    window.removeEventListener('pointerdown', handlePointerDown);
+  }
+}
+
+window.addEventListener('pointerdown', handlePointerDown);
+
+export function isMouseUsed() {
```

Comment:
One of the nice things about Javascript exports is that they are referential - this has the nice effect of being able to export dynamic variables from a module.

This means that we don't have to have `isMouseUsed` as a function, instead it can be initialized as:

```
export let isMouseUsed = localStorage.getItem('mouseUsed') === 'true';
```